src/js/components/ClinPhen/PhenopacketDisplay/MedicalActionsView.tsx

```diff
@@ -18,34 +19,38 @@ import type { Quantity } from '@/types/clinPhen/measurement';
 import type { OntologyTerm } from '@/types/ontology';
 import type { Procedure } from '@/types/clinPhen/procedure';
 import type { TimeInterval } from '@/types/clinPhen/shared';
-import { useTranslatedTableColumnTitles } from '@/hooks/useTranslatedTableColumnTitles';
+import type { ConditionalDescriptionItem } from '@/types/descriptions';
+
 import { useTranslationFn } from '@/hooks';
 
 const { Text } = Typography;
 
 export const ProcedureComponent = ({ procedure }: { procedure: Procedure }) => {
-  const ProcedureItems: DescriptionsProps['items'] = [
+  const ProcedureItems: ConditionalDescriptionItem[] = [
     {
       key: 'code',
       label: 'medical_actions.code',
       children: <OntologyTermComponent term={procedure.code} />,
+      isVisible: procedure.code,
     },
     {
       key: 'bodySite',
       label: 'medical_actions.body_site',
       children: <OntologyTermComponent term={procedure.body_site} />,
+      isVisible: procedure.body_site,
     },
     {
       key: 'performed',
       label: 'medical_actions.performed',
       children: <TimeElementDisplay element={procedure.performed} />,
+      isVisible: procedure.performed,
     },
   ];
   return <TDescriptions bordered column={1} size="small" items={ProcedureItems} />;
 };
 
 const TreatmentComponent = ({ treatment }: { treatment: Treatment }) => {
-  const DOSE_INTERVAL_COLUMNS = useTranslatedTableColumnTitles<DoseInterval>([
+  const DOSE_INTERVAL_COLUMNS = [
```

Comment:
type hint perhaps; if this is not a constant (i.e., uses params) it should not be in ALL_CAPS